### ISSUE TITLE
Time series plot....Wind Lake....database issue or out-of-date packages issue?

### ISSUE THREAD

**Issue by sbmarsh-USGS** [USGS-R/lakeReport#44]
Wind Lake is an example (424915088083900). Not sure if it's incorrect data in the database or what's going on.  I'm putting a picture of last year's time series plot and the one I just generated.

![image](https://cloud.githubusercontent.com/assets/10731242/18170322/8f3c7aca-7023-11e6-84e1-0ae1b17cd7c4.png)

![image](https://cloud.githubusercontent.com/assets/10731242/18170336/9d37a33e-7023-11e6-993a-bc4c0c8422cc.png)

**Comment by sbmarsh-USGS** [comment]
TSI plot (bottom one) - way off.  2015 vs. 2014 shown below.

![image](https://cloud.githubusercontent.com/assets/10731242/18568377/94d9ddac-7b62-11e6-9770-f64d8ff8b9d9.png)

![image](https://cloud.githubusercontent.com/assets/10731242/18568391/a48fdb20-7b62-11e6-8582-5a84e99a5645.png)

**Comment by sbmarsh-USGS** [comment]
I fixed the data in the database...added meter depths where there were only feet and added chlorophyll values that were missing.  Not sure what the issue is with bottom plot, though.

**Comment by lindsaycarr** [comment]
@sbmarsh-USGS when I run the TSI plot using lakeReport functions, 2014 and 2015 do not differ much. Can you double check that the equations I've implemented are correct?

```r
siteNumber <- '424915088083900'
plotNames <- 'qwtimeseries'

wy <- 2014
makeReports(siteNumber, wy, plotNames, 'word')

wy <- 2015
makeReports(siteNumber, wy, plotNames, 'word')
```

**Comment by sbmarsh-USGS** [comment]
I ran what you suggested and yes, in R, the 2014 and 2015 look similar, but they aren't similar to the 2014 plot from SPLUS, which leads me to believe the equations aren't correct.  How do I check these?  I don't see them in the qwtimeseries.Rmd.

**Comment by sbmarsh-USGS** [comment]
Just figured it out.  Your equations use LOG instead of LN.

**Comment by lindsaycarr** [comment]
Ahhhh OK. Thanks for finding that - I'll edit those and let you know when the changes are merged.

**Comment by lindsaycarr** [comment]
***OR*** it would be a great opportunity for you to edit and submit your changes since we hope that minor edits can be made on your own at some point in the future! Let's chat about it on Monday.

**Comment by sbmarsh-USGS** [comment]
Yes, I would love to practice doing that!  Let me know when you're available to chat.

**Comment by lindsaycarr** [comment]
take a look at this page: http://www.statmethods.net/management/functions.html

It looks like `log()` is natural log and `log10()` is the "normal log" function...

**Comment by sbmarsh-USGS** [comment]
Hmmm...confused, because I ran Log on our existing data and got the same numbers that you plotted, which are different from the LN stuff, so we'll need to talk.

**Comment by sbmarsh-USGS** [comment]
Closed by #48
